14th-team5-iOS/Bibbi/Sources/Presentation/Notification/ViewControllers/NotificationViewController.swift

```diff
@@ -0,0 +1,277 @@
+//
+//  NotificationViewController.swift
+//  App
+//
+//  Created by 마경미 on 04.01.25.
+//
+
+import Core
+import UIKit
+
+import RxSwift
+import RxDataSources
+
+final class NotificationFooterView: UITableViewHeaderFooterView {
+    static let id = "notificationFooterView"
+    
+    private let leftLine: UIView = UIView()
+    private let label: BBLabel = .init(.caption, textColor: .gray500)
+    private let rightLine: UIView = UIView()
```

Comment:
```suggestion
    private let leftLineView: UIView = UIView()
    private let labelView: BBLabel = .init(.caption, textColor: .gray500)
    private let rightLineView: UIView = UIView()
```

이렇게 View를 붙이는건 어떤가요?

Reply:
넵